.github/workflows/CD.yaml

```diff
@@ -194,8 +190,8 @@ jobs:
         run: cd dataland-frontend && npm install && npm run testgithubenvironment --url="https://${{ secrets.TARGETSERVER_URL }}"
         env:
           ELECTRON_EXTRA_LAUNCH_ARGS: "--ignore-connections-limit=dataland.com,test.dataland.com,dev1.dataland.com,dev2.dataland.com"
-          CYPRESS_PREPOPULATE_TIMEOUT_S: ${{ github.event.inputs.useInMemoryStorage && 180 || 900 }}
-          CYPRESS_PREVISIT_TIMEOUT_S: ${{ github.event.inputs.useInMemoryStorage && 180 || 1200 }}
+          CYPRESS_PREPOPULATE_TIMEOUT_S: ${{ 900 }}
+          CYPRESS_PREVISIT_TIMEOUT_S: ${{ 1200 }}
```

Comment:
900/1200 are needed for EuroDaT - as this took ages. 
For any internal service, the shorter values (180/180) should do the job.

Reply:
As 180 is anyways set in `dataland-frontend/cypress.env.json`, we should remove this variable everywhere. What do you thing?

Reply:
For the current state where we do not use EuroDaT, we would not need to set them here. The variables CYPRESS_PREVISIT_TIMEOUT_S and CYPRESS_PREPOPULATE_TIMEOUT_S seem not to be used right now, but PREVISIT_TIMEOUT_S and PREPOPULATE_TIMEOUT_S via Cypress.env. I assume the prefix is automatically added before the lookup then? Are the constants from these file automatically applied? Then there would be more redundant constants defined, like AWAIT_PREPOPULATION_RETRIES, I'll look that up and update the code.

Reply:
It seems reasonable to remove one ore the other pair. I'll remove the CYPRESS_* variables for now. Although they will be probably more handy when dealing with EuroDaT again. What irritates me: Have they even been of use recently? Or have they been overwritten by the ones from cypress.env.json all the time.

Reply:
The other variables defined in that file make it appear that, the variables in cypress.env.json are only applied (if at all) on the developer machine. In that case we would need to set them in the docker-compose, right?